lib/model.js

```diff
@@ -3375,6 +3375,7 @@ function _setIsNew(doc, val) {
  * trip to MongoDB.
  *
  * Mongoose will perform casting on all operations you provide.
+ * The only exception [setting the `update` operator for `updateOne` or `updateMany` to a pipeline](https://www.mongodb.com/docs/manual/reference/method/db.collection.bulkWrite/#updateone-and-updatemany): Mongoose does **not** cast update pipelines.
```

Comment:
```suggestion
 * The only exception is [setting the `update` operator for `updateOne` or `updateMany` to a pipeline](https://www.mongodb.com/docs/manual/reference/method/db.collection.bulkWrite/#updateone-and-updatemany): Mongoose does **not** cast update pipelines.
```

i think the `is` was accidentally deleted, or am i just reading it wrong?